data/data-pipeline/data_pipeline/tests/base/config.yaml

```diff
@@ -1,10 +0,0 @@
-name: Template
```

Comment:
These files are all from an older approach to dataset schemas that's out of date.